packages/node_modules/@webex/webex-core/test/unit/spec/services/service-catalogs.js

```diff
@@ -190,45 +190,154 @@ describe('webex-core', () => {
-        filteredHosts = serviceCatalogs.filterCluster();
+        filteredHosts = serviceCatalogs.filterLocal();
 
         assert.equal(filteredHosts.length, serviceCatalogs.hosts.length);
         assert.deepEqual(filteredHosts, serviceCatalogs.hosts);
       });
 
-      it('should return only service hosts in the specific cluster', () => {
-        filteredHosts = serviceCatalogs.filterCluster(hostFixtureA.id);
+      it('should return only local hosts when called with true', () => {
+        filteredHosts = serviceCatalogs.filterLocal(true);
 
-        assert.equal(filteredHosts.length, 1);
-        assert.equal(filteredHosts[0].id, hostFixtureA.id);
+        assert.equal(filteredHosts.length, localHosts.length);
+        assert.isTrue(filteredHosts.every(
+          (host) => host.local === true
+        ));
+      });
+
+      it('should return only hosts remote hosts when called with false', () => {
+        filteredHosts = serviceCatalogs.filterLocal(false);
+
+        assert.equal(filteredHosts.length, remoteHosts.length);
+        assert.isTrue(filteredHosts.every(
+          (host) => host.local === false
+        ));
+      });
+    });
+
+    describe('#filterPriority()', () => {
+      let filteredHosts;
+      let priorityHosts;
+
+      beforeEach('generate the service host class objects', () => {
+        serviceCatalogs.load(
+          ServiceCatalogs.mapRemoteCatalog({
+            catalog: SERVICE_CATALOGS[0],
+            ...fixture
+          })
+        );
+
+        priorityHosts = [
+          fixture.hostCatalog['example-service-a'][0],
+          fixture.hostCatalog['example-service-b'][0],
+          fixture.hostCatalog['example-service-c'][0]
+        ];
+      });
+
+      it('should return all hosts when called without params', () => {
+        filteredHosts = serviceCatalogs.filterPriority();
+
+        assert.equal(filteredHosts.length, serviceCatalogs.hosts.length);
+        assert.deepEqual(filteredHosts, serviceCatalogs.hosts);
+      });
+
+      it('should return only priority hosts when called with true', () => {
+        filteredHosts = serviceCatalogs.filterPriority(true);
+
+        assert.equal(filteredHosts.length, priorityHosts.length);
+      });
+
+      it('should not return inactive hosts when called with true', () => {
+        filteredHosts = serviceCatalogs.filterPriority(true);
+        filteredHosts[0].setStatus({failed: true});
+
+        const failedHost = filteredHosts[0];
+
+        filteredHosts = serviceCatalogs.filterPriority(true);
+
+        assert.notInclude(filteredHosts, failedHost);
       });
 
-      it('should return service hosts for an array of clusters', () => {
-        filteredHosts = serviceCatalogs.filterCluster(
-          [hostFixtureA.id, hostFixtureB.id]
+      it('should return all hosts when called with false', () => {
+        filteredHosts = serviceCatalogs.filterPriority(false);
+
+        assert.equal(filteredHosts.length, serviceCatalogs.hosts.length);
```

Comment:
Same here, we don't need both assertions

Reply:
Addressing this as well as all other cases that it appears as a redundant assertion.